app/carpool/views.py

```diff
@@ -116,6 +141,7 @@ def start_geojson():
                 'leave_time': pool.leave_time.isoformat(),
                 'return_time': pool.return_time.isoformat(),
                 'driver_gender': escape(pool.driver.gender),
+                'approx_location': approx_location
```

Comment:
Can we change this to `is_approximate_location`. No need for abbreviation and adding the "is" makes it clear.what the bool means.